Dockerfile

```diff
@@ -65,17 +65,126 @@ RUN curl -sL https://deb.nodesource.com/setup_10.x | bash - \
            # Packages to install \
            libsasl2-dev freetds-bin build-essential sasl2-bin \
            libsasl2-2 libsasl2-dev libsasl2-modules \
-           default-libmysqlclient-dev apt-utils curl rsync netcat locales  \
+           dirmngr apt-utils curl rsync netcat locales  \
            freetds-dev libkrb5-dev libssl-dev libffi-dev libpq-dev git \
            nodejs gosu sudo \
     && apt-get autoremove -yqq --purge \
     && apt-get clean \
     && rm -rf /var/lib/apt/lists/*
 
+# Install MySQL client from Oracle repositories (Debian installs mariadb)
+RUN KEY="A4A9406876FCBD3C456770C88C718D3B5072E1F5" \
+    && GNUPGHOME="$(mktemp -d)" \
+    && export GNUPGHOME \
+    && for KEYSERVER in $(shuf -e \
+            ha.pool.sks-keyservers.net \
+            hkp://p80.pool.sks-keyservers.net:80 \
+            keyserver.ubuntu.com \
+            hkp://keyserver.ubuntu.com:80 \
+            pgp.mit.edu) ; do \
+          gpg --keyserver "${KEYSERVER}" --recv-keys "${KEY}" && break || true ; \
+       done \
+    && gpg --export "${KEY}" > /etc/apt/trusted.gpg.d/mysql.gpg \
+    && gpgconf --kill all \
+    rm -rf "${GNUPGHOME}"; \
+    apt-key list > /dev/null \
```

Comment:
What's the purpose of this line?

Reply:
I think I can remove that line. It is more for debugging to check if the key was imported than anything else.

Reply:
Oh - rather than doing ` > /etc/apt/trusted.gpg.d/mysql.gpg` could we do `apt-key add -` so that we'd get an error if something went wrong?

Reply:
I will try.